From b63a8d6f74afa394dec8039803cd1435fcd9bcaf Mon Sep 17 00:00:00 2001
From: chcoutur <chcoutur@student.42.fr>
Date: Wed, 28 Sep 2022 16:46:44 +0200
Subject: [PATCH] Update kernel name

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a4f71076cacb..05a312797191 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 0
 SUBLEVEL = 0
-EXTRAVERSION = -rc4
+EXTRAVERSION = -rc4-thor_kernel
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
-- 
2.37.3

